index.js

```diff
@@ -46,6 +47,10 @@ module.exports = (source, destination, options = {}) => {
 		let files;
 		try {
 			files = await globby(source, options);
+
+			if (options.ignoreJunk) {
+				files = files.filter(file => junk.not(file.substring(file.lastIndexOf('/') + 1, file.length)));
```

Comment:
Use `path.basename()`. Your check will fail on Windows and some other situations.